helmfile/01-applications.yaml

```
@@ -235,18 +235,18 @@ releases:
 {{ end }}
 
 
-#Elasticsearch-prometheus-exporter
-- name: elasticsearch-exporter
+# prometheus-elasticsearch-exporter
+- name: prometheus-elasticsearch-exporter
   namespace: elastic-system
   labels:
-    app: elasticsearch-exporter
-  chart: stable/elasticsearch-exporter
-  version: 3.3.0
+    app: prometheus-elasticsearch-exporter
+  chart: prometheus-community/prometheus-elasticsearch-exporter
+  version: 4.0.0
```

If you want you can bump it to :)
```suggestion
  version: 4.0.1
```

---

Sure, fixed that too.